lec01.tex

```diff
@@ -141,14 +141,6 @@ \subsection{Equivalent Definitions of prBPP via Amplification}
   $\delta_0$ and $\delta_1$ ensure that $t = \poly(n)$.
 \end{proof}
 
-\begin{corollary}\label{cor:bpp-small-random}
-  For all $\eps > 0$, the class $\prBPP$ can be equivalently defined with an
-  error of $2^{T^\eps - T}$ instead of 1/3, where $T = T(n)$ is the number of
-  random bits.
-
-  In other words, $2^{T^\eps}$ out of the $2^T$ random bit-strings are exceptional.
-\end{corollary}
-
```

Comment:
I think this was supposed to be moved in 159186f58fcb81760fdb2c372238cc91da47eb0f.